sources/src/main/java/com/google/solutions/jitaccess/core/adapters/SlackAdapter.java

```diff
@@ -0,0 +1,52 @@
+package com.google.solutions.jitaccess.core.adapters;
```

Comment:
Add Apache header

(I know it's annoying, but we have to do this)

Reply:
No worries, added!